src/main/java/com/arpnetworking/clusteraggregator/ClusterStatusCache.java

```diff
@@ -102,7 +108,17 @@ public void onReceive(final Object message) throws Exception {
         if (!getSender().equals(getSelf())) {
             // Public messages
             if (message instanceof ClusterEvent.CurrentClusterState) {
```

Comment:
I just want to validate that we have achieved a reasonable solution here. Who and when is the ClusterEvent.CurrentClusterState message being sent to this actor?

Each such message will result in a new unit of work being created and samples logged to disk. Is the cadence above consistent and appropriate for such a use case?